Watching for file changes with StatReloader
Performing system checks...

System check identified some issues:

WARNINGS:
todos.Todo: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the TodosConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.

System check identified 1 issue (0 silenced).
February 17, 2023 - 11:40:08
Django version 4.1.7, using settings 'todoApp.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.
Invalid HTTP_HOST header: '137.117.107.181:8000'. You may need to add '137.117.107.181' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/dist-packages/django/utils/deprecation.py", line 135, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/dist-packages/django/http/request.py", line 152, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '137.117.107.181:8000'. You may need to add '137.117.107.181' to ALLOWED_HOSTS.
Bad Request: /
[17/Feb/2023 11:41:25] "GET / HTTP/1.1" 400 62323
Invalid HTTP_HOST header: '137.117.107.181:8000'. You may need to add '137.117.107.181' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/dist-packages/django/utils/deprecation.py", line 135, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/dist-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/dist-packages/django/http/request.py", line 152, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '137.117.107.181:8000'. You may need to add '137.117.107.181' to ALLOWED_HOSTS.
Bad Request: /favicon.ico
[17/Feb/2023 11:41:26] "GET /favicon.ico HTTP/1.1" 400 62160
/root/projects/django-todo/todoApp/settings.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified some issues:

WARNINGS:
todos.Todo: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the TodosConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.

System check identified 1 issue (0 silenced).
February 17, 2023 - 11:42:30
Django version 4.1.7, using settings 'todoApp.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.
[17/Feb/2023 11:42:41] "GET / HTTP/1.1" 302 0
[17/Feb/2023 11:42:41] "GET /todos HTTP/1.1" 301 0
[17/Feb/2023 11:42:42] "GET /todos/ HTTP/1.1" 200 3515
[17/Feb/2023 11:42:42] "GET /static/css/style.css HTTP/1.1" 200 225
Not Found: /favicon.ico
[17/Feb/2023 11:42:43] "GET /favicon.ico HTTP/1.1" 404 2576
[17/Feb/2023 14:48:24] "GET /todos/ HTTP/1.1" 200 3515
[17/Feb/2023 14:48:25] "GET /static/css/style.css HTTP/1.1" 304 0
[17/Feb/2023 14:57:08] "GET / HTTP/1.1" 302 0
